### PR TITLE
refactor!: remove SlowComponentLoader and ReflectiveInjector.

### PR DESCRIPTION
**Pull request by ykmnkmi**
BREAKING CHANGE: remove `SlowComponentLoader`, `ReflectiveInjector` classes with dependent `runAppLegacy`, `runAppLegacyAsync` and `bootstrapStatic` functions.

### CODE REVIEW

**Comment by GZGavinZhao** [comment]
I'm trying to enforce [conventional commit](https://www.conventionalcommits.org) now, so if you want to squash & merge this PR as one big commit, please rename the PR title (which will become the commit title) when you submit this as ready for review. Otherwise, when submitting, please rebase/edit/squash/force push or whatever you like to ensure individual commits follow conventional commit guidelines.

File: _tests/test/di/directive_test.dart [hunk 1]
```diff
@@ -4,6 +4,19 @@ import 'package:test/test.dart';
 
 import 'directive_test.template.dart' as ng;
 
+Object missingServicFactory(Object willNotBeCalled) {
```
Comment (GZGavinZhao) [inline _tests/test/di/directive_test.dart:7]: `missingServiceFactory`?

Comment (ykmnkmi) [reply _tests/test/di/directive_test.dart]: updated.

**Review by GZGavinZhao — APPROVED** [review]
Looks good to me. Thank you!

**Comment by GZGavinZhao** [comment]
Just one small thing, in the function I commented on, did you mean to type `missingServiceFactory` instead of `missingServicFactory`?

**Comment by ykmnkmi** [comment]
Yes.

**Pull request merged**